cmd/root.go

```diff
@@ -91,6 +91,7 @@ func init() {
 	rootCmd.PersistentFlags().Int32("max-ejection-percentage", -1, "maximal percentage of hosts ejected via outlier detection. Set to >=0 to activate outlier detection in envoy.")
 	rootCmd.PersistentFlags().Int64("host-selection-retry-attempts", -1, "Number of host selection retry attempts. Set to value >=0 to enable")
 	rootCmd.PersistentFlags().String("retry-on", "5xx", "default comma-separated list of retry policies")
+	rootCmd.PersistentFlags().String("tracing-provider", "", "HTTP Connection Manager tracing provider block to include")
```

Comment:
Might be good to document that only Zipkin is supported at the moment somewhere.

Maybe we can add a section on "Tracing" to the readme for that?

Reply:
Good idea, I'll add something to the docs now